tensorflow/lite/micro/micro_profiler.h

```diff
@@ -72,6 +77,17 @@ class MicroProfiler {
   uint32_t end_ticks_[kMaxEvents];
   int num_events_ = 0;
 
+  struct TicksPerTag {
+    const char* tag;
+    uint32_t ticks;
+  };
+  // In practice, the number of tags will be much lower than the number of
+  // events. But it is theoretically possible that each event to be unique and
+  // hence we allow total_ticks_per_tag to have kMaxEvents entries.
+  TicksPerTag total_ticks_per_tag[kMaxEvents];
+
+  int FindExistingOrNextPosition(const char* tagName);
```

Comment:
tag_name: https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Done. Missed the header file last time.